API/SSW.Rewards.Domain/Entities/Achievement.cs

```diff
@@ -16,6 +16,8 @@ public class Achievement : Entity
 
         public bool IconIsBranded { get; set; }
 
+        public bool IsDeleted { get; set; }
```

Comment:
maybe add a PBI to the backlog:
- move this to the abstract Entity
- add global query filters to auto exclude where deleted (note: there is an IgnoreGlobalQueryFilter extension method if you need to show deleted data)

Reply:
Done: https://github.com/SSWConsulting/SSW.Rewards/issues/275